sdk/core/azure-core/samples/test_example_async.py

```diff
@@ -53,29 +53,29 @@ async def req():
 async def test_example_asyncio():
 
     request = HttpRequest("GET", "https://bing.com")
-    policies = [UserAgentPolicy("myuseragent"), AsyncRedirectPolicy()]
+    policies : Iterable[Union[AsyncHTTPPolicy, SansIOHTTPPolicy]] = [UserAgentPolicy("myuseragent"), AsyncRedirectPolicy()]
     # [START asyncio]
     from azure.core.pipeline.transport import AsyncioRequestsTransport
 
     async with AsyncPipeline(AsyncioRequestsTransport(), policies=policies) as pipeline:
         response = await pipeline.run(request)
     # [END asyncio]
-    assert pipeline._transport.session is None
+    assert pipeline._transport.session is None  # type: ignore
     assert isinstance(response.http_response.status_code, int)
 
 
 @pytest.mark.asyncio
 async def test_example_aiohttp():
 
     request = HttpRequest("GET", "https://bing.com")
-    policies = [UserAgentPolicy("myuseragent"), AsyncRedirectPolicy()]
+    policies : Iterable[Union[AsyncHTTPPolicy, SansIOHTTPPolicy]] = [UserAgentPolicy("myuseragent"), AsyncRedirectPolicy()]
```

Comment:
I think `black` will be opinionated on the extra whitespace

```suggestion
    policies: Iterable[Union[AsyncHTTPPolicy, SansIOHTTPPolicy]] = [UserAgentPolicy("myuseragent"), AsyncRedirectPolicy()]
```